### PR TITLE
Remove simple dependencies

### PR DESCRIPTION
**Pull request by ai**
Dependencies cleaning started by @NickKaramoff. This PR will reduce the size of project dependencies and make `node_modules` of everyone smaller by cleaning unnecessary docs and `package.json` of simple dependencies.

- [x] [`indent-string`](https://www.npmjs.com/package/indent-string) because it is a simple function, which we use in a single place. The space of the package was taken mostly by `package.json`, docs and unnecessary for our case checks.
- [x] [`figures`](https://www.npmjs.com/package/figures) because we use only a few symbols. I moved them (both main and fallback versions) to a separated file.

### CODE REVIEW

**Comment by ai** [comment]
@cenk1cenk2  `test 10.x` CI failed because of:

```
error ts-node@10.0.0: The engine "node" is incompatible with this module. Expected version ">=12.0.0". Got "10.24.1"
error Found incompatible module.
```

Seems like it is an issue non-relevant to this PR changes.

**Comment by cenk1cenk2** [comment]
Thank you this is just fits on the exact path I want to take. If only we can move indent string function to utils even though it is only used in the default renderer it will be perfect. I will merge it hopefully tomorrow. Thank you for your work thst you have put in to this.

**Comment by ai** [comment]
> If only we can move indent string function to utils even though it is only used in the default renderer it will be perfect.

Done.

Thanks for the fast replies.

File: src/utils/indent-string.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export function indentString (string: string, count: number): string {
+  return string.replace(/^(?!\s*$)/gm, ' '.repeat(count))
+}
```
Comment (NickKaramoff) [inline src/utils/indent-string.ts:3]: This file doesn't have a newline at the end of it... is this a big problem @cenk1cenk2?

**Review by NickKaramoff — APPROVED** [review]

File: src/utils/indent-string.ts (hunk 1 above)
Comment (ai) [reply src/utils/indent-string.ts]: Fixed e04c796

**Comment by ai** [comment]
@cenk1cenk2 I also exported `figures` because `lint-staged` teams asked a way to sync symbols 

https://github.com/okonet/lint-staged/pull/981#issuecomment-848766712

**Comment by cenk1cenk2** [comment]
Okay. That is no problem. Maybe today I will not have a oppurtinity to merge it since I am swamped with work but I will merge it asap.

**Comment by cenk1cenk2** [comment]
Sincerely appriciate for your time on this @ai and @NickKaramoff, thank you again.

**Pull request merged**